translations/stripes-acq-components/en.json

```diff
@@ -96,10 +96,6 @@
   "piece.pieceFormat.other": "Other",
   "piece.pieceStatus.expected": "Expected",
   "piece.pieceStatus.received": "Received",
-  "piece.pieceStatus.late": "Late",
```

Comment:
Why did you delete translations?

Reply:
I haven't. Let me check it

Reply:
It seems, that initially I created the branch from master, and changes were kept after I switched develop. Good catch.